cmd/geth/config.go

```diff
@@ -107,12 +109,17 @@ func defaultNodeConfig() node.Config {
 	return cfg
 }
 
+func defaultDashboardConfig() dashboard.Config {
+	return dashboard.DefaultConfig
+}
```

Comment:
You don't really need a method for this. `defaultNodeConfig` has its own method because it modifies the `node.DefaultConfig` with custom stuff. But if you are only returning the `dashboard.DefaultConfig` here, you might as well use that in line 122.